<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTreeView" name="view"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuActions_2">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionAdd_Artist"/>
    <addaction name="actionAdd_Album"/>
    <addaction name="actionAdd_Song"/>
    <addaction name="separator"/>
    <addaction name="actionRemove_Artist"/>
    <addaction name="actionRemove_Album"/>
    <addaction name="actionRemove_Song"/>
   </widget>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="menuActions_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Artist"/>
   <addaction name="actionAdd_Album"/>
   <addaction name="actionAdd_Song"/>
   <addaction name="separator"/>
   <addaction name="actionRemove_Artist"/>
   <addaction name="actionRemove_Album"/>
   <addaction name="actionRemove_Song"/>
   <addaction name="separator"/>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="separator"/>
   <addaction name="actionClose"/>
  </widget>
  <action name="actionAdd_Artist">
   <property name="text">
    <string>Add Artist</string>
   </property>
  </action>
  <action name="actionAdd_Album">
   <property name="text">
    <string>Add Album</string>
   </property>
  </action>
  <action name="actionAdd_Song">
   <property name="text">
    <string>Add Song</string>
   </property>
  </action>
  <action name="actionRemove_Artist">
   <property name="text">
    <string>Remove Artist</string>
   </property>
  </action>
  <action name="actionRemove_Album">
   <property name="text">
    <string>Remove Album</string>
   </property>
  </action>
  <action name="actionRemove_Song">
   <property name="text">
    <string>Remove Song</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
